Packs/Gamma/Integrations/Gamma/Gamma.py

```diff
@@ -0,0 +1,321 @@
+
+    note = ''
+    if v_list['response'][0]['violation_id'] != int(minimum_violation):
+        note += f'Violation with the minimum_violation ID does not exist. Showing violations pulled from the next available ID: {v_list["response"][0]["violation_id"]} \r'
+
+    human_readable = note
+    for i in v_list['response']:
+        violation_id = i['violation_id']
+        human_readable += f'### Violation {i["violation_id"]} \r' \
+                          f'|Violation ID|Status|Timestamp|Dashboard URL|User|App Name| \r' \
+                          f'|---|---|---|---|---|---| \r' \
+                          f'| {violation_id} | {i["violation_status"]} | {timestamp_to_datestring(i["violation_event_timestamp"]*1000)} | {i["dashboard_url"]} | {i["user"]} | {i["app_name"]} | \r'
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix="GammaViolation",
+        outputs_key_field="violation_id",
+        outputs=v_list,
+        raw_response=v_list
+    )
+
+
+def get_violation_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    :type client: Client
+    :param client: Gamma client
+
+    :param args: all command arguments, usually passed from demisto.args()
+        args['name'] is used as input name
+
+    :return:
+        A CommandResults object that is then passed to return_results
+
+    :rtype: ``CommandResults``
+    """
+
+    violation_id = args["violation"]
+
+    if not int(violation_id) >= 1:
+        raise ValueError("violation must be greater than 0")
+
+    violation = client.get_violation(violation_id)
+
+    if violation['response'][0]['violation_id'] != int(violation_id):
+        return "Violation with this ID does not exist."
+
+    human_readable = ''
+    for i in violation['response']:
+        human_readable += f'### Violation {i["violation_id"]} \r' \
+                          f'|Violation ID|Status|Timestamp|Dashboard URL|User|App Name| \r' \
+                          f'|---|---|---|---|---|---| \r' \
+                          f'| {i["violation_id"]} | {i["violation_status"]} | {timestamp_to_datestring(i["violation_event_timestamp"] * 1000)} | {i["dashboard_url"]} | {i["user"]} | {i["app_name"]} | \r'
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix="GammaViolation",
+        outputs_key_field="violation_id",
+        outputs=violation,
+        raw_response=violation
+    )
+
+
+def update_violation_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    :type client: Client
+    :param client: Gamma client
+
+    :param args: all command arguments, usually passed from demisto.args()
+        args['name'] is used as input name
+
+    :return:
+        A CommandResults object that is then passed to return_results
+
+    :rtype: ``CommandResults``
+    """
+
+    violation = args["violation"]
+    status = args["status"].upper()
+    notes = args["notes"]
```

Comment:
Since this argument is not required, then this access can fail.
You can chagne it to `.get('notes', '')`.